api/schemas/Place.yaml

```diff
@@ -0,0 +1,33 @@
+type: object
+description: Represents a location relative to which an organization wants to track its users movements.
+required:
+  - id
+  - name
+  - position
+  - data
+properties:
+  id:
+    $ref: "./GenericId.yaml#"
+  name:
+    type: string
+  # TODO check validity of this data type compared to leaflet's polygons
+  polygon:
+    type: array
+    description: An array of points that constitutes the perimeter of a place
+    minItems: 3
+    items:
+      type: object
+      description: Angular coordinates that represent a point on Earth's surface.
+      required:
+        - latitude
+        - longitude
+      properties:
+        latitude:
+          type: number
+          format: double
+        longitude:
+          type: number
+          format: double
```

Comment:
Qui bisognerebbe imporre che siano coordinate sferiche valide (cioè, la longitudine deve essere in [-180, +180] e la latitudine in [-90, +90]), però si può aggiungere più avanti.